accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java

```diff
@@ -73,7 +73,7 @@ protected void doPost(final HttpServletRequest req, final HttpServletResponse re
 
         RequestParameters reqParams = RequestParameters.fromRequest(req, acInstallationService);
         LOG.info("Received POST request to apply AC Tool config with configurationRootPath={} basePaths={}",
-                reqParams.configurationRootPath, reqParams.basePaths);
+                escapeHtml4(reqParams.configurationRootPath), reqParams.basePaths);
```

Comment:
why do you escape for HTML parsers in a log? A log line is never evaluated as HTML IMHO